src/containers/Tenant/utils/queryTemplates.ts

```diff
@@ -46,10 +46,10 @@ export const createAsyncReplicationTemplate = () => {
     return `CREATE OBJECT secret_name (TYPE SECRET) WITH value="secret_value";
 
 CREATE ASYNC REPLICATION my_replication
-FOR \`/remote_database/table_name\` AS "local_table_name" --[, \`/remote_database/another_table_name\` AS "another_local_table_name" ...]
+FOR \`/remote_database/table_name\` AS \`local_table_name\` --[, \`/remote_database/another_table_name\` AS \`another_local_table_name\` ...]
 WITH (
-    CONNECTION_STRING=\`grpcs://mydb.ydb.tech:2135/?database=/remote_database\`,
-    TOKEN_SECRET_NAME = "secret_name",
+    CONNECTION_STRING="grpcs://mydb.ydb.tech:2135/?database=/remote_database",
+    TOKEN_SECRET_NAME = "secret_name"
```

Comment:
Such template will be valid as it is and replication will be created (just insert and run), though it will create useless not working replication